controls/radpdfviewer/features/find.md

```diff
@@ -0,0 +1,147 @@
+
+{{region radpdfviewer-find_2}}
+
+    SearchResult searchResult = this.pdfViewer.FindPrevious(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    Dim searchResult As SearchResult = Me.pdfViewer.FindPrevious(sampleText, textSearchOptions)
+{{endregion}}
+
+## SearchResult
+
+The **SearchResult** class holds the result from a specific search command. This class exposes several properties and methods:
+* **NotFound**: A static property which gets a default SearchResult instance when there is no search result found.
+* **Range**: Gets the result text range position.
+* **Result**: Gets the result as text.
+* **ToString**: Returns a string that represents the current object.
+
+#### __[C#] Example 4: Check if a match was found__
+
+{{region radpdfviewer-find_3}}
+
+    if (searchResult != SearchResult.NotFound)
+    {
+        TextRange searchResultRange = searchResult.Range;
+        string searchResultAsText = searchResult.Result;
+    }
+{{endregion}}
+
+#### __[VB.NET] Example 4: Check if a match was found__
+
+{{region radpdfviewer-find_3}}
+
+    If searchResult <> SearchResult.NotFound Then
+        Dim searchResultRange As TextRange = searchResult.Range
+        Dim searchResultAsText As String = searchResult.Result
+    End If
+{{endregion}}
+
+## TextSearchOptions
+
+The **TextSearchOptions** class holds all possible search options when invoking finding. The class has various constructors that can be used depending on the necessities. Their main difference is they are receiving different amount of parameters according to the selected options.
+* **TextSearchOptions(bool caseSensitive)**
+* **TextSearchOptions(bool caseSensitive, bool useRegularExpression)**
+* **TextSearchOptions(bool caseSensitive, bool useRegularExpression, bool wholeWordsOnly)**
+
+This class exposes a static property:
+* **Default**: Gets the default text search options.
+
+    #### __[C#] Example 5: The default search options__
+    {{region radpdfviewer-find_4}}
+    
+        TextSearchOptions textSearchOptions = 
+            new TextSearchOptions(caseSensitive: false, useRegularExpression: false, wholeWordsOnly: false);
+    {{endregion}}
+    
+    #### __[VB.NET] Example 5: The default search options__
+    
+    {{region radpdfviewer-find_4}}
+    
+        Dim textSearchOptions As TextSearchOptions = 
+            New TextSearchOptions(caseSensitive:=False, useRegularExpression:=False, wholeWordsOnly:=False)
+    {{endregion}}
+
+several non-static boolean properties:
+* **UseRegularExpression**: Gets or sets the value indication whether the regular expression option is enabled.
+* **CaseSensitive**: Gets or sets the value indication whether the case sensitive option is enabled.
+* **WholeWordsOnly**: Gets or sets the value indication whether the whole words only option is enabled.
```

Comment:
* **UseRegularExpression**: Gets or sets a value indicating whether a regular expression should be used for searching.
* **CaseSensitive**: Gets or sets a value indicating whether the search should be case sensitive.
* **WholeWordsOnly**: Gets or sets a value indicating whether only whole words should be matched.